UI/users/preferences.html

```diff
@@ -100,7 +103,8 @@
                                   options = user.cssfiles
                                   default_values = [ user.prefs.stylesheet ]
                                   text_attr = 'file'
-                                  value_attr = 'file'
+                                  value_attr = 'file',
```

Comment:
None of the other lines ends in a comma. Why should this one?

Reply:
Fixed